geotrek/api/v2/views/flatpages.py

```diff
@@ -40,7 +41,11 @@ class MenuItemRetrieveView(RetrieveAPIView):
 class MenuItemTreeView(GenericAPIView):
     # from https://stackoverflow.com/questions/21112302/how-to-serialize-hierarchical-relationship-in-django-rest
     serializer_class = api_serializers.MenuItemSerializer
-    queryset = flatpages_models.MenuItem.objects.filter(depth=1)
+    queryset = (
+        flatpages_models.MenuItem.objects
+        .filter(depth=1)
+        .filter(platform__in=[MenuItem.PLATFORM_CHOICES.ALL, MenuItem.PLATFORM_CHOICES.WEB])
```

Comment:
for maintenability, I think it is better to exclude MOBILE than filter ALL and WEB (in case of some other options are added in future)

Reply:
I agree, done.